templates/adding_a_new_model/tests/tf-seq-2-seq-bart-tokenizer.json

```diff
@@ -1,11 +1,11 @@
 {
   "modelname": "NewTFENCDEC",
   "uppercase_modelname": "NEW_TF_ENC_DEC",
-  "lowercase_modelname": "new_tf_enc_dec",
+  "lowercase_modelname": "new_tf_enc_dec_template",
```

Comment:
@LysandreJik - we were actually never testing the PT and TF encoder-decoder templates since the names didn't match the `*template*` regex. Fixing it here and also fixing all the model templates so that all tests pass